README.md

```diff
@@ -1,13 +1,38 @@
 # Eppo SDK for Golang
```

Comment:
have no idea how this got here, but this is simply from this PR: https://github.com/Eppo-exp/golang-sdk/pull/11